bokeh/__init__.py

```diff
@@ -10,10 +14,9 @@
     __version__ = get_versions()['version']
     del get_versions
 
-
 _notebook_loaded = None
 
-def load_notebook(resources=None, verbose=False, force=False, skip=False):
+def load_notebook(resources=None, verbose=False, skip=False):
```

Comment:
Oh I see I forget the usage be outside the diff. It's not often used, but it should also be renamed. Can you change it to `show_banner` or something similar (and update the template as well)? And then `skip` could be used to skip loading in `__init__.py` (might be sometimes useful)